packages/platform-express/adapters/express-adapter.ts

```diff
@@ -76,6 +80,10 @@ export class ExpressAdapter extends AbstractHttpAdapter {
     return this.use(handler);
   }
 
+  public isHeadersSent(response: any): boolean {
+    return response.headersSent;
```

Comment:
```suggestion
    return !!response.headersSent;
```

in my tests, `response.headersSents` ends up being `undefined`